code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -128,7 +128,15 @@
 	can_cut_open = 0
 	icon_state = "jacksandal"
 	item_color = "jacksandal"
-
+	
+/obj/item/clothing/shoes/jackboots/noisy
+	name = "heavy jackboots"
+	desc = "Old, heavy versions of the standard Nanotrasen-issue Security combat boots. Pick up that can."
+	
+/obj/item/clothing/shoes/jackboots/noisy/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/jackboot1.ogg' = 1, 'sound/effects/jackboot2.ogg' = 1), 50, falloff_exponent = 20)
```

Comment:
The squeak component isn't ideal here because it registers all sorts of unneeded signals, equip/drop/crossed etc. However at the moment, it looks like there's not really a better way to do this.

Reply:
I think it's fine given it would be a BSH reward.